content/information-distribution/source.md

```diff
@@ -4,13 +4,48 @@ weight: 2
 ---
 
 A source is an entity holding the responsibility of publishing information
-through the Matrix room. Before publishing any information, a source **must**
-register itself as such on the room, including data on the organism or
-individual operating the source, the publication language and the publication
-name space (2️⃣: a Matrix event type), along with one or more public signature
-verification keys that will be used to assert the integrity of the information
-it publishes, and the signing algorithm used to generate the signature of its
-articles (`ed25519`, `hmac-sha256`, `hmac-sha512`, etc.).
+through the Matrix room. Each source **must** be a registered Matrix user. A
+Matrix user **must not** register more than one source.
+
+Before publishing any information, a source **must** register itself as such on
+the room. This **must** be done through the publication of a state event of the
+`network.informo.source` type. The event's state key **must** be the ID of the
+source's Matrix user. The content of this event **must** be provided using the
+following model:
+
+|      Parameter      |    Type    | Req. |                                                        Description                                                       |
+| ------------------- | ---------- | :--: | ------------------------------------------------------------------------------------------------------------------------ |
+| `name`              | `string`   |  x   | Name of the source.                                                                                                      |
+| `origin`            | `string`   |  x   | The company or individual maintaining this source.                                                                       |
+| `langs`             | `[lang]`   |  x   | Languages of the source's publications.                                                                                  |
```

Comment:
I'd rather go for `l10n`, which is more obvious than `langs` (and I don't really like the naming of `localized_sources` because it refers to sub-sources as sources).